_docs/tooling/maven.md

```diff
@@ -91,5 +92,20 @@ Here is the list of properties supported by the goal:
     increase compilation time, sometimes can make code even slower; 
     recommended for WebAssembly.
 * **targetType** &ndash; what code to generate. Following options are supported:
-  * *JAVASCRIPT*
-  * *WEBASSEMBLY*
\ No newline at end of file
+  * *JAVASCRIPT* to generate JavaScript
+  * *WEBASSEMBLY* to generate WebAssembly.
```

Comment:
`C` option as well

Reply:
Certainly. That is what I could not simply see only from the one Maven plugin class. Later yesterday I realized.